lib/utils/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by kugajjar on 6/4/15.
+ */
+'use strict';
+
+module.exports = function utils() {
```

Comment:
You can export an object so you don't need to call `utils` as a function in the rest of your code.

```
module.exports = {
  generateRandomNumber: function (length) {
    return ...;
  }
};
```

Then when you're using it...

```
var utils = require('./utils');
utils.generateRandomNumber(1);
```